src/foam/u2/view/RichChoiceView.js

```diff
@@ -374,7 +377,7 @@ foam.CLASS({
                   return Promise.all(promiseArray).then((resp) => {
                     var index = 0;
                     return this.E().forEach(sections, function(section) {
-                      this
+                      this.addClass('setAbove')
```

Comment:
I don't think this works. addClass 'setAbove' will set the class to just 'setAbove', but the css above is for ^setAbove which gets expanded to include the package and class name.